packages/core/src/feedback.ts

```diff
@@ -1,15 +1,12 @@
-import type { EventHint, FeedbackEvent, SendFeedbackParams } from '@sentry/types';
+import type { FeedbackEvent, SendFeedbackOptions, SendFeedbackParams } from '@sentry/types';
 import { dropUndefinedKeys } from '@sentry/utils';
 import { getClient, getCurrentScope } from './currentScopes';
 
 /**
  * Send user feedback to Sentry.
  */
-export function captureFeedback(
-  feedbackParams: SendFeedbackParams,
-  hint: EventHint & { includeReplay?: boolean } = {},
-): string {
-  const { message, name, email, url, source, associatedEventId } = feedbackParams;
+export function captureFeedback(feedbackParams: SendFeedbackParams, hint: SendFeedbackOptions = {}): string {
```

Comment:
m: Can we please leave` hint be typed as `EventHint & ...`, e.g. `EventHint & SendFeedbackOptions`? Makes it more explicit that this is just a regular hint, not something feedback special.

Reply:
np!